### PR TITLE
add more content #3

### PR DESCRIPTION
**Pull request by xtremi**
does this look ok?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,5 +2,9 @@
   <head><title>Welcome to potionfy</title></head>
   <body>
     <h1>Welcome to potionfy! Enjoy!</h1>
+    <h2>something 1</h2>
+    <p>bla bla bla</p>
+    <h2>something 2</h2>
+    <p>bla bla bla</p>
```
Comment (xtremi2) [inline index.html]: Not acceptable! Please change ASAP!

**Review by xtremi2 — CHANGES_REQUESTED** [review]
asap

File: index.html (hunk 1 above)
Comment (xtremi) [reply index.html]: I see

**Comment by xtremi** [comment]
I'm on it.

**Review by xtremi2 — APPROVED** [review]
perfect

**Pull request merged**